### DISCUSSION TITLE
Search Bug when typing same thing twice

### DISCUSSION THREAD

**Discussion by bnunAG** [requarks/wiki#5103] [Error / Bug Report] [answered]
I noticed this on my test environment and also on the webpage - https://docs.requarks.io/guide/intro

If you type "test" or anything in the search, hit escape, and then type test again you get "No pages matching your query."

I've tested this in Edge, Edge (Dev), and Chrome. Thought maybe it was a browser issue, then I thought maybe it was our test environment or an Elasticsearch issue. Was surprised to see it on the website.

**Comment by vincentDcmps** [comment]
HI same issue on a postgress database here.
I note that in both case graphQL request return same result so something in the client traitment don't print two time the same result perhaps

**Comment by NGPixel** [comment, marked as answer]
This is fixed since 2.5.282
